lib/matplotlib/__init__.py

```diff
@@ -1214,19 +1214,31 @@ def rc_file_defaults():
                          if k not in STYLE_BLACKLIST})
 
 
-def rc_file(fname):
+def rc_file(fname, *, use_default_template=True):
     """
     Update rc params from file.
 
     Style-blacklisted rc params (defined in
     `matplotlib.style.core.STYLE_BLACKLIST`) are not updated.
+
+    Parameters
+    ----------
+    fname : str
+        Name of file parsed for matplotlib settings.
+
+    use_default_template : bool
+        If True, initialize with default parameters before updating with those
+        in the given file. If False, the current configuration persists
+        and only the parameters specified in the file are updated.
+
     """
     # Deprecation warnings were already handled in rc_params_from_file, no need
     # to reemit them here.
     with warnings.catch_warnings():
         warnings.simplefilter("ignore", mplDeprecation)
         from .style.core import STYLE_BLACKLIST
-        rc_from_file = rc_params_from_file(fname)
+        rc_from_file = rc_params_from_file(fname,
+                                    use_default_template=use_default_template)
```

Comment:
That's not PEP8: I don't understand why our tests are not picking this up. Do you know what's going on here?

Reply:
No idea. I could go for

~~~
        rc_from_file = rc_params_from_file(
            fname, use_default_template=use_default_template)
~~~

if you want, but IMHO that's less readable. If flake8 is fine as is, I'm too :wink:

Reply:
That would indeed be flake8 compliant (I'd be more comfortable with adding pep8-compliant lines).
In the meantime, I'll try to figure out where (and why) we are ignoring PEP8 errors on this file.

Reply:
Changed.

AFAIK, `.flake8` is the only thing that influences flake8. Maybe the globally ignored E121 is allowing this.